src/platform-implementation-js/dom-driver/gmail/views/gmail-thread-row-view.ts

```diff
@@ -317,53 +317,39 @@ class GmailThreadRowView {
     this._userView = userView;
   }
 
-  getCounts(): Counts {
-    let counts = this._counts;
-
-    if (!counts) {
-      const recipientsElement = querySelector(this._elements[0], 'td div.yW');
-
-      if (this._driver.isUsingSyncAPI()) {
-        const draftCount = recipientsElement.querySelectorAll('.boq').length;
-        const messageCountMatch = recipientsElement.querySelector('.bx0');
-        const messageCount =
-          messageCountMatch && messageCountMatch.innerHTML
-            ? +messageCountMatch.innerHTML
-            : draftCount
-            ? 0
-            : 1;
-        counts = this._counts = {
-          messageCount,
-          draftCount,
-        };
-      } else {
-        const [preDrafts, drafts] = recipientsElement.innerHTML.split(
-          /<font color=[^>]+>[^>]+<\/font>/
-        );
-        const preDraftsWithoutNames = preDrafts.replace(
-          /<span\b[^>]*>.*?<\/span>/g,
-          ''
-        );
-        const messageCountMatch = preDraftsWithoutNames.match(/\((\d+)\)/);
-        const messageCount = messageCountMatch
-          ? +messageCountMatch[1]
-          : preDrafts
-          ? 1
-          : 0;
-        const draftCountMatch = drafts && drafts.match(/\((\d+)\)/);
-        const draftCount = draftCountMatch
-          ? +draftCountMatch[1]
-          : drafts != null
-          ? 1
-          : 0;
-        counts = this._counts = {
-          messageCount,
-          draftCount,
-        };
-      }
+  #getCounts(): Counts {
```

Comment:
This method is used by `GmailThreadRowView.prototype.getVisibleDraftCount` and `GmailThreadRowView.prototype.getVisibleMessageCount`. Both seem to be used by the public interface `ThreadRowView`.

The other place the latter two methods above are used are in `ThreadRowIdentifier`, which isn't part of the public API. Those lines are here:

https://github.com/InboxSDK/InboxSDK/blob/c76c2b978c80eea4012d6f65755601be8bf9e420/src/platform-implementation-js/dom-driver/gmail/gmail-driver/thread-row-identifier.ts#L39 and https://github.com/InboxSDK/InboxSDK/blob/c76c2b978c80eea4012d6f65755601be8bf9e420/src/platform-implementation-js/dom-driver/gmail/gmail-driver/thread-row-identifier.ts#L80